client/tst/redux/reducers/DiningHallInformationReducers.test.js

```diff
@@ -0,0 +1,61 @@
+import { diningHallsList } from '../../../app/redux/reducers/DiningHallInformationReducers'
+import * as types from '../../../app/redux/actions/types'
+import { allActions } from '../../../app/redux/actions';
+import stateProperties from '../../../app/redux/lib/stateProperties';
+import { locationsResponse } from '../../config/responseObjects/locationsResponse';
+
+
+const initialState = {
+    isLoading: true,
+    hasError: false,
+    errorMessage: undefined,
+    data: undefined,
+    dataObject: undefined
+}
+
+
+describe ('DiningHallsInformationReducers', () => {
+
+    it('should handle GET_ALL_DINING_HALLS_INFORMATION_REQUEST', () => {
+        const requestAction = {
+            type: types.GET_ALL_DINING_HALLS_INFORMATION_REQUEST
+        };
+        expect(diningHallsList(initialState, requestAction)).toEqual({
+            ...initialState, 
+            hasError: false, 
+            isLoading: true
+        });
+    });
+
+    it('should handle GET_ALL_DINING_HALLS_INFORMATION_FAILURE', () => {
+        const failureAction = {
+            type: types.GET_ALL_DINING_HALLS_INFORMATION_FAILURE,
+            errorMessage: 'errorMessage'
+        };
+        expect(diningHallsList(initialState, failureAction)).toEqual({
+            ...initialState,
+            data: undefined,
+            dataObject: undefined,
+            isLoading: false, // set isLoading to false so our UI knows we're not still loading :(
```

Comment:
I notice the copypasta LOL (that comment is still there)

Reply:
lol removed